README.md

```diff
@@ -46,6 +52,11 @@ class Service::Foo < Service::Base
   # Receives a config hash containing :identifier => value pairs for each input field.
   # Returns an array 2-tuple containing a boolean and a response message.
   # The boolean should be true if the data in the config was verified, otherwise false.
+  #
+  # You can also raise errors out of this method, but the response with return to users
```

Comment:
First sentence is odd: "response with return to users".  Did you mean "response we return to users"?